WME_Image_Overlays.user.js

```diff
@@ -888,26 +859,22 @@ function setTranslations(translations) {
 }
 
 // Create a tab and possibly receive a previous tab to restore (usually in case of a mode change)
-function addTab(recoveredTab) {
```

Comment:
{recoveredTab} is no longer needed as there is no longer another way addTab is called. Previously, it was called from the imperial / metric state change.